src/components/chart-preview/ChartPreview.tsx

```diff
@@ -52,6 +55,42 @@ export const ActualChart = ({
     useSaveCsvData(csv, "chart-data");
   };
 
+  const onFigureDownloadClick = (e: any) => {
+    //TODO rework getting parent node
+    const fullFigureNode =
+      e.target.parentNode.parentNode.parentNode.parentNode.parentNode;
+
+    let figureCloneNode = fullFigureNode.cloneNode(true);
+    let emptyNode = document.createElement("div");
+    emptyNode.setAttribute(
+      "style",
+      "position: absolute; opacity: 0; pointerEvents: none; width: 1280px;",
+    );
+    figureCloneNode.setAttribute("style", "background-color: white;");
+    emptyNode.append(figureCloneNode);
+    fullFigureNode.append(emptyNode);
```

Comment:
this append and append back feels confusing to me (but that might be my backend background showing!)

Would it not be simpler to:
```

let emptyNode = document.createElement("div");
emptyNode.append(fullFigureNode.cloneNode(true));
// set all the attributes
fullFigureNode.append(emptyNode);
```
You could potentially avoid figureCloneNode altogether?

Oh hang on, figureCloneNode is what we're downloading. Why do we need to append these new elements to the DOM at all then?